### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by atakanserifoglu**

### CODE REVIEW

**Comment by atakanserifoglu** [comment]
!cond_accept

**Comment by tech-extremists** [comment]
!cond_accept

File: .github/workflows/pr_agent.yml [hunk 1]
```diff
@@ -18,3 +18,5 @@ jobs:
         env:
           OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          github_action_config.auto_review: "true" # enable\disable auto review
+          github_action_config.auto_describe: "true"
```
Comment (tech-extremists) [inline .github/workflows/pr_agent.yml:22]:
```suggestion
          github_action_config.auto_review: "true" # enable\disable auto review please
          github_action_config.auto_describe: "true"
```

**Comment by tech-extremists** [comment]
!cond_accept

**Review by tech-extremists — APPROVED** [review]
Approved as the changes align with the suggestions.

**Pull request closed without merging**